Telerik.JustMock/Core/Context/MockingContext.cs

```diff
@@ -71,18 +71,56 @@ public static MocksRepository ResolveRepository(UnresolvedContextBehavior unreso
 			return LocalMockingContextResolver.ResolveRepository(unresolvedContextBehavior);
 		}
 
-		public static void RetireRepository()
+#if !PORTABLE
+		private static readonly object retireRepositoryounterSync = new object();
```

Comment:
There is a typo here. I guess the variable should be retireRepositoryCounterSync